webpack/components/pf3Table/formatters/selectionHeaderCellFormatter.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import TableSelectionHeaderCell from '../components/TableSelectionHeaderCell';
 
-export default (selectionController, label) => (
+export default (selectionController, label, hasPermission) => (
```

Comment:
I don't think we should carry the hasPermission variable name into this part of the code since it's not related to permissions at all at this point. Would selectionEnabled be a better name?